src/Extensions/Diagnostics.HealthChecks/ServiceCollectionExtensions.cs

```diff
@@ -15,10 +17,7 @@ namespace Microsoft.Omex.Extensions.Diagnostics.HealthChecks
 	/// </summary>
 	public static class ServiceCollectionExtensions
 	{
-		/// <summary>
-		/// Register types required for Omex health check
-		/// </summary>
-		public static IHealthChecksBuilder AddServiceFabricHealthChecks(this IServiceCollection serviceCollection)
```

Comment:
I think we would need to keep the method signature as it is, to avoid breaking existing clients and just add an alternative method to register new publisher

Reply:
Restored old signature